src/PowerOfThree.java

```diff
@@ -0,0 +1,17 @@
+public class PowerOfThree {
+    public static boolean isPowerOfThree(int n) {
+        if(n <= 0) {
+            return false;
+        }
+
+        while(n % 3 == 0 && n > 3) {
```

Comment:
You can change condition to while(n % 3 == 0 && n >= 3), then return can be:
return n == 1